controllers/policyset/policyset_controller.go

```diff
@@ -234,44 +241,56 @@ func (r *PolicySetReconciler) processPolicySet(ctx context.Context, plcSet *poli
 	return needsUpdate
 }
 
-// getStatusMessage returns a message listing disabled, deleted and pending policies
-func getStatusMessage(disabledPolicies []string, pendingPolicies []string, deletedPolicies []string) string {
-	statusMessage := "All policies are reporting status"
+// getStatusMessage returns a message listing disabled, deleted and policies with no status
+func getStatusMessage(
+	disabledPolicies []string,
+	unknownPolicies []string,
+	deletedPolicies []string,
+	pendingPolicies []string,
+) string {
+	statusMessage := ""
 	separator := ""
+	allReporting := true
 
-	if len(disabledPolicies) > 0 {
-		statusMessage = fmt.Sprintf("Disabled policies: %s", strings.Join(disabledPolicies, ", "))
+	if len(pendingPolicies) > 0 {
+		allReporting = false
+		statusMessage += fmt.Sprintf(separator+"Policies awaiting pending dependencies: %s",
```

Comment:
Optional: It's not needed here since it's the first encountered if statement.
```suggestion
		statusMessage += fmt.Sprintf("Policies awaiting pending dependencies: %s",
```